airbyte-cdk/bulk/toolkits/extract-cdc/src/main/kotlin/io/airbyte/cdk/read/cdc/Debezium.kt

```diff
@@ -85,18 +85,19 @@ value class DebeziumRecordValue(val wrapped: JsonNode) {
     }
 }
 
-/** Debezium Engine input. */
-data class DebeziumInput(
-    val properties: Map<String, String>,
-    val state: DebeziumState,
-    val isSynthetic: Boolean,
-)
-
-/** Debezium Engine output, other than records of course. */
-data class DebeziumState(
```

Comment:
These two types just weren't used that much and it seems that getting rid of them does improve things.